services/gundeck/Makefile

```diff
@@ -22,17 +20,23 @@ guard-%:
 	    exit 1; \
 	fi
 
-default: clean install
+default: fast
 
 .PHONY: init
 init:
-	mkdir -p dist
+	mkdir -p dist ../../dist
 
 .PHONY: install
 install: init
-	stack install --pedantic --test --bench --no-run-benchmarks --local-bin-path=dist
-	cp "$(shell stack path --dist-dir)/build/$(NAME)-tests/$(NAME)-tests" dist/
-	cp "$(shell stack path --dist-dir)/build/$(NAME)-bench/$(NAME)-bench" dist/
+	stack install --pedantic --test --bench --no-run-benchmarks --local-bin-path=../../dist
+	cp "$(shell stack path --dist-dir)/build/$(NAME)-tests/$(NAME)-tests" ../../dist/
+	cp "$(shell stack path --dist-dir)/build/$(NAME)-bench/$(NAME)-bench" ../../dist/
+
+.PHONY: fast
+fast: init
+	stack install --pedantic --test --bench --no-run-benchmarks --local-bin-path=../../dist --fast
```

Comment:
Can we remove `--bench --no-run-benchmarks` so that it matches the `fast` target from other services? Otherwise it causes a recompilation since the flags don't match the ones on other services.

Reply:
Good catch, thanks!